packages/dev/docs/src/StateTable.js

```diff
@@ -24,7 +24,7 @@ export function StateTable({properties, showOptional}) {
   }
 
   return (
-    <table className={`${tableStyles['spectrum-Table']} ${tableStyles['spectrum-Table--quiet']} ${styles.propTable}`}>
+    <table className={`${tableStyles['spectrum-Table']} ${styles.propTable}`}>
```

Comment:
Got rid of spectrum quiet variant for props tables. The border all the way around seems to give a bit more structure to the page IMO.

Reply:
Looks so much better